modules/creating-infra-machines.adoc

```diff
@@ -3,10 +3,17 @@
 // * post_installation_configuration/cluster-tasks.adoc
 
 [id="creating-infra-machines_{context}"]
-= Creating infrastructure machines
+= Creating infrastructure machines with a dedicated configuration
```

Comment:
Why call it a "dedicated" configuration? It's no more dedicated than any other part I think.  This is just to describe how to use the MCP approach.